gulpfile.js

```diff
@@ -39,7 +39,7 @@ function gulp_test() {
     return gulp.src(tests)
     .pipe(mocha({
         log: true,
-        timeout: 60000,
+        timeout: 1200000,
```

Comment:
120s is 2 minutes and thats just per test giving it a bit more time to download the docker container + start it up and run the npm install. If you do a fresh npm install from scratch you'll see that alone takes 60s on average.